### ISSUE TITLE
Stacktrace does not work properly

### ISSUE THREAD

**Issue by stefandd** [aardappel/lobster#52]
var a = 1
let b = 0
print a / b

---- result with head rev, both MSVC and MinGW/MSYS:
stackunwind.lobster(3): VM error: division by zero
in function: __top_level_expression -> stackunwind.lobster(3)
globals:
   a = 0

---- expected:
stackunwind.lobster(3): VM error: division by zero
in function: __top_level_expression -> stackunwind.lobster(3)
globals:
   a = 1

**Comment by aardappel** [comment]
Thanks for reporting! Yes the concept of a "global" had changed (they're now in a top level function) and the stack trace dumping code has not been updated. This should mostly fix it (now shows `a = 1`): https://github.com/aardappel/lobster/commit/dd98377cf40e4baed3387a2159cd9e4e81e41400

The code is still not ideal, but certainly an improvement :)

**Comment by stefandd** [comment]
I do not think this is fixed since the following code produces no info on the globals:
```
let n = 1
let a, b = 2, 3
print n / 0
```
produces ```bug2.lobster(3): VM error: division by zero
in function: __top_level_expression -> bug2.lobster(3)```
